.storybook/main.ts

```diff
@@ -1,17 +1,15 @@
-import type { StorybookConfig } from '@storybook/experimental-nextjs-vite';
+import type { StorybookConfig } from '@storybook/nextjs-vite';
 
 const config: StorybookConfig = {
     stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
     addons: [
         'storybook-addon-apollo-client',
         '@storybook/addon-links',
-        '@storybook/addon-essentials',
         '@storybook/addon-onboarding',
         '@chromatic-com/storybook',
-        '@storybook/experimental-addon-test',
```

Comment:
This package was for running unit tests directly inside the storybook app. I removed it because we aren't utilizing it as far as I know.